apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -1,8 +1,14 @@
+import { Icon } from "~/libs/components/components.js";
 import { combineClassNames } from "~/libs/helpers/helpers.js";
+import { useMemo } from "~/libs/hooks/hooks.js";
+import { type IconName } from "~/libs/types/types.js";
 
 import styles from "./styles.module.css";
 
 type Properties = {
+	icon?: IconName;
+	isDisabled?: boolean;
+	isPressed?: boolean;
```

Comment:
it's not pressed, it should be a different variant of button, maybe something like ghost
```suggestion
	icon?: IconName;
	isDisabled?: boolean;
```